packages/eas-cli/src/submissions/android/ServiceAccountSource.ts

```diff
@@ -80,3 +116,38 @@ async function askForServiceAccountPathAsync(): Promise<string> {
   });
   return filePath;
 }
+
+async function confirmDetectedPathAsync(path: string): Promise<boolean> {
+  Log.log(
+    `A valid Google Service Account JSON key has been found in \n   ${chalk.underline(path)}`
+  );
+  const { confirmed } = await promptAsync({
+    name: 'confirmed',
+    type: 'confirm',
+    message: 'Would you like to use this file?',
+    initial: true,
+  });
+
+  Log.addNewLineIfNone();
+  return confirmed;
+}
+
+async function fileIsGoogleServicesAsync(path: string): Promise<boolean> {
+  try {
+    const jsonFile = await fs.readJson(path);
+    return jsonFile.type === 'service_account';
+  } catch (e) {
+    return false;
+  }
+}
+
+/**
+ * asynchronous array filter
+ * @param arr array to filter
+ * @param predicate a predicate function to run asynchronously
+ * @returns a promise resolving to a filtered array
+ */
+const filterAsync = async <T>(
+  arr: T[],
+  predicate: (arg: T, index?: number, array?: T[]) => Promise<boolean>
+) => Promise.all(arr.map(predicate)).then(results => arr.filter((_v, index) => results[index]));
```

Comment:
Should I move it to somewhere in `src/utils` dir? Where exactly? 🤔

Reply:
src/utils/filterAsync seems fine to me!